client/src/components/Card.spec.js

```diff
@@ -2,8 +2,9 @@ import { render, screen } from '@testing-library/react'
 import Card from './Card'
 
 describe('Card', () => {
-  it('renders its children', () => {
-    render(<Card>Hello world</Card>)
-    expect(screen.getByText('Hello world')).toBeInTheDocument()
+  it('renders its text and author', () => {
+    render(<Card text={'What is your favorite color?'} author={'John Doe'} />)
+    expect(screen.getByText('What is your favorite color?')).toBeInTheDocument()
+    expect(screen.getByText('– John Doe')).toBeInTheDocument()
```

Comment:
The dash might not be as imporant here.

```suggestion
    expect(screen.getByText('John Doe', {exact: false})).toBeInTheDocument()
```